config/locales/en.yml

```diff
@@ -257,7 +257,13 @@ en:
           overseas_recognition: "This is the subject you qualified with."
       eligible_itt_subject: "Which subject did you do your %{qualification} in?"
       teaching_subject_now: "Do you teach %{eligible_itt_subject} now?"
-      itt_academic_year: "In what academic year did you %{start_or_complete} your %{qualification}?"
+      itt_academic_year:
```

Comment:
Tying translations to the way you've broken down a grammatical structure probably isn't a good idea because that structure might not be compatible with other languages.

I know we only have English but then you could say why do we have translations at all. I'd change this to make sure it's compatible with other languages, see example directly below your change.

Reply:
@nomasprime so you just want 4 entries?

We already do this elsewhere in the file, so not sure why being called out now?

I think we are using translations more for the simplicity of we only have to change them in one place